.azure-devops/templates/run-tests.yml

```diff
@@ -34,9 +34,15 @@ steps:
 
   - template: set-testenv-sentinel.yml
 
+  # Fix for DNS resolution issues: https://github.com/actions/virtual-environments/issues/798
```

Comment:
When is run-tests.yml used over run-tests-parallel.yml? Can it be removed?

Reply:
Currently merge.yml uses run-tests. But, it can surely use the run-tests-parallel. I will update the reference and remove it.